intelmq/lib/bot.py

```diff
@@ -1,66 +1,64 @@
-import re
-import sys
+# -*- coding: utf-8 -*-
+from __future__ import print_function, unicode_literals
```

Comment:
The `__future__` package brings some changes that are mandatory in future python versions, see [docs](https://docs.python.org/2.7/library/__future__.html). The import of `print_function` deletes the print statement and imports the print function, the `unicode_literals` makes all strings like `'foobar'` to unicode strings by default `u'foobar'`. Testing for unicode in quite essential for the logger.

Both features are standard in python3, so this file is compatible to python3 in regard of unicode (there are also other changes, but these are important here).